FizzBuzz/FizzBuzz/Program.cs

```diff
@@ -23,46 +21,42 @@ public static int insertIfExistsWord(int at, ArrayList arr, string specifiedChar
             return exists;
         }
 
-        public static void print(int number, ArrayList arr)
+        public static string print(int number, ArrayList arr)

```

Comment:
this name is now a bit misleading, because it's not printing anything. Maybe something like `printToString` would be more suited?